lib/ble.toit

```diff
@@ -343,13 +491,24 @@ class Device:
     finally:
       ble_scan_stop_ resource_group_
 
-STARTED_EVENT_        ::= 1 << 0
-COMPLETED_EVENT_      ::= 1 << 1
-DISCOVERY_EVENT_      ::= 1 << 2
-CONNECTED_EVENT_      ::= 1 << 3
-CONNECT_FAILED_EVENT_ ::= 1 << 4
+  client_connected:
```

Comment:
The methods name here are bit weird. Maybe `wait_for_client_connected` would be better? At least I find the example hard to understand where you use this method. Are you telling the system that the client has connected or are you waiting for something?

Reply:
changed the method names to the suggested wait_for_ variants.